synapse_rock/rockcraft.yaml

```diff
@@ -129,4 +126,18 @@ parts:
         plugin: dump
         source: attributemaps
         organize:
-            login_ubuntu.py: /usr/local/attributemaps/login_ubuntu.py
\ No newline at end of file
+            login_ubuntu.py: /usr/local/attributemaps/login_ubuntu.py
+    synapse-start:
+        plugin: dump
+        after:
+          - synapse
+        source: .
+        # start.py is extracted from
+        # https://github.com/matrix-org/synapse/blob/develop/docker/start.py
```

Comment:
You may want to point to https://github.com/element-hq/synapse/blob/develop/docker/start.py as the other repo is a public archive.

Reply:
Removed ;)